pages/vi/nextcloud-tor.md

```diff
@@ -3,11 +3,12 @@
 ## Objectives
 
 * Practice using ssh and Treehouses Remote  
+* Practice running docker container
```

Comment:
"Docker" instead of "docker"

Reply:
Thanks for catching this. Fixed in the commit.